extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -958,21 +959,25 @@ export class AmpLightboxGallery extends AMP.BaseElement {
       duration = this.getTransitionDurationFromElements_(srcImg, targetImg);
       motionDuration = MOTION_DURATION_RATIO * duration;
       // Prepare the actual image animation.
-      imageAnimation = prepareImageAnimation({
-        styleContainer: this.getAmpDoc().getHeadNode(),
-        transitionContainer: this.getAmpDoc().getBody(),
-        srcImg,
-        targetImg,
-        srcImgRect: undefined,
-        targetImgRect: undefined,
-        styles: {
-          'animationDuration': `${motionDuration}ms`,
-          // Matches z-index for `.i-amphtml-lbg`.
-          'zIndex': 2147483642,
-        },
-        keyframesNamespace: undefined,
-        curve: TRANSITION_CURVE,
-      });
+      try {
+        imageAnimation = prepareImageAnimation({
+          styleContainer: this.getAmpDoc().getHeadNode(),
+          transitionContainer: this.getAmpDoc().getBody(),
+          srcImg,
+          targetImg,
+          srcImgRect: undefined,
+          targetImgRect: undefined,
+          styles: {
+            'animationDuration': `${motionDuration}ms`,
+            // Matches z-index for `.i-amphtml-lbg`.
+            'zIndex': 2147483642,
+          },
+          keyframesNamespace: undefined,
+          curve: TRANSITION_CURVE,
+        });
+      } catch(e) {
+        reportError(e);
+      }
```

Comment:
Just double checking, so bailing out of this basically results in us fading in the lightbox background and carousel but just without the translation, right? Do we need a different animation (e.g. curves) for this or is it fine?

Reply:
This results in us fading in the background, with the carousel suddenly appearing. I don't think we need to worry too much about the transition here since this should only occur due to a bug, which we should fix.